src/kata1.fizzBuzz.js

```diff
@@ -1,3 +1,6 @@
-const fizzBuzz = (number) => {};
+const fizzBuzz = (number) => {
+    return (number%3===0 && number%5!=0)? 'Fizz': (number%3!=0 && number%5===0)? 'Buzz': (number%3===0 && number%5===0)? 'FizzBuzz': number
+};
```

Comment:
This works, but is not at all easy to read! Its considered bad practice to nest multiple ternary expressions for this reason. Instead you could use the same logic but break it up a bit so its quicker and easier to understand:
```
const fizzBuzz = (number) => {
    if (number % 3 === 0 && number % 5 !== 0) return "Fizz";
    if (number % 3 !== 0 && number % 5 === 0) return "Buzz";
    if (number % 3 === 0 && number % 5 === 0) return "FizzBuzz";
    return number;
};
```
Much easier to work out whats going on. Also remember that you should use `!==` and not `!=` for not equal to expressions